iceoryx_examples/callbacks/README.md

```diff
@@ -165,16 +165,17 @@ void heartbeatCallback(iox::popo::UserTrigger*)
 }
 ```
 
-The `onSampleReceivedCallback` is more complex. We first acquire the received
-sample and check which subscriber signaled the event by acquiring the subscriber's
+The `onSampleReceivedCallback` is more complex. We first acquires all the received
+samples and checks which subscriber signaled the event by acquiring the subscriber's
```

Comment:
```suggestion
samples and check which subscriber signaled the event by acquiring the subscriber's
```

Reply:
I shouldn't type in the evening :sweat_smile: